src/Analyzers/CSharp/CodeFixes/CSharpCodeFixes.projitems

```diff
@@ -87,6 +87,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)UseDeconstruction\CSharpUseDeconstructionCodeFixProvider.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UseIsNullCheck\UseIsNullCheckHelpers.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UsePatternMatching\CSharpAsAndMemberAccessCodeFixProvider.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)UseThrowExpression\UseThrowExpressionCodeFixProvider.cs" />
```

Comment:
This was always a c# only fixer.  moved to C# project.  (needed this since we look at particular syntax pieces to move trivia as well).